app/Http/Controllers/CustomFieldsController.php

```diff
@@ -169,11 +167,35 @@ public function update(CustomFieldRequest $request, $id)
         $field->name = e($request->get("name"));
         $field->element = e($request->get("element"));
         $field->field_values = e($request->get("field_values"));
-        $field->field_encrypted = e($request->get("field_encrypted", 0));
         $field->user_id = Auth::user()->id;
         $field->help_text = $request->get("help_text");
         $field->show_in_email = $request->get("show_in_email", 0);
 
+        /**
+         * Encrypted field should not be shown in emails
+         */
+        if ($request->input('field_encrypted') == 1) {
+            $field->show_in_email = 0;
+        }
+
+        /**
+         * Change the encryption status of the field
+         */
+        if ($request->input('field_encrypted') != $field->field_encrypted) {
```

Comment:
I'm not sure how I feel about allowing anyone with custom fields access to be able to turn off encryption. I could see some potential security issues there. We might need to add another Custom Fields permissions.

Reply:
(Possibly related to #5810)